wordpress-mobile/gutenberg-mobile
# Feature/lists

Refs #208 
~Gutenberg PR https://github.com/WordPress/gutenberg/pull/14249~
New Gutenberg PR https://github.com/WordPress/gutenberg/pull/14636

![simulator screen shot - iphone xr - 2019-03-05 at 22 29 37](https://user-images.githubusercontent.com/651601/53842255-d86a4c00-3f96-11e9-96e2-e88deba750bd.png)

This PR implements the list block by only changing components and **not touching** the code of the original web block.

This achieved by improving the implementation of the block-edit context in order to send the isSelected and onFocus props to the RichText components.

For the moment it doesn't implement the indent and outdent buttons, because of a bug in Aztec for iOS that doesn't allow split to work properly when lists are indented.

If testing on iOS please run `yarn preios` to make sure you get the right version of Aztec. 

How to test:
 - Start the demo app 
 - Check the list display properly and you can interact with them and change type
 - Add new list block to the post
 - Check that the list block works properly.
